validation/views.py

```diff
@@ -512,3 +532,43 @@ def user_is_expert(user):
                 return True
 
     return False
+
+
+def prep_phrase_data(request, phrases):
+    # The _segment_card needs a dictionary of recordings
+    # in order to properly display search results
+    recordings = {}
+    forms = {}
+    for phrase in phrases:
+        recordings[phrase] = phrase.recordings
+        if request.method == "POST" and int(request.POST.get("phrase_id")) == phrase.id:
+            forms[phrase.id] = FlagSegment(
+                request.POST, initial={"phrase_id": phrase.id}
+            )
+        else:
+            forms[phrase.id] = FlagSegment(initial={"phrase_id": phrase.id})
+
+    return recordings, forms
+
+
+def save_issue(data, user):
+    phrase_id = data["phrase_id"]
+    issues = data["issues"]
+    other_reason = data["other_reason"]
+    comment = data["comment"]
+
+    phrase = Phrase.objects.get(id=phrase_id)
+
+    new_issue = Issue(
+        phrase=phrase,
+        other="other" in issues,
+        bad_cree="bad_cree" in issues,
+        bad_english="bad_english" in issues,
+        bad_recording="bad_rec" in issues,
+        comment=comment,
+        other_reason=other_reason,
+        created_by=user,
+        created_on=datetime.datetime.now(),
```

Comment:
You can just delete the default in the `Issue` model if you use this every time.